coderd/workspaceagents.go

```diff
@@ -464,15 +465,29 @@ func (api *API) dialWorkspaceAgentTailnet(r *http.Request, agentID uuid.UUID) (*
 	}
 
 	conn, err := tailnet.NewConn(&tailnet.Options{
-		Addresses: []netip.Prefix{netip.PrefixFrom(tailnet.IP(), 128)},
-		DERPMap:   derpMap,
-		Logger:    api.Logger.Named("tailnet"),
+		Addresses:      []netip.Prefix{netip.PrefixFrom(tailnet.IP(), 128)},
+		DERPMap:        derpMap,
+		Logger:         api.Logger.Named("tailnet"),
+		BlockEndpoints: true,
```

Comment:
Why do we set BlockEndpoints now? I'm sure there's a reason, just wondering. 😄

Reply:
This is a mistake! Fantastic catch 🥴🥴🥴